src/main_thread/tracks_store/tracks_store.ts

```diff
@@ -651,23 +654,27 @@ export default class TracksStore extends EventEmitter<ITracksStoreEvents> {
     }
 
     // Check that we're not both disabling audio and video here
-    if (storedSettings === null) {
-      const periodItem = getPeriodItem(this._storedPeriodInfo, period.id);
-      if (periodItem !== undefined) {
-        const hasNoTrackAtAll = ["audio" as const, "video" as const].every(
-          (ttype) => periodItem[ttype].storedSettings === null,
-        );
-        if (hasNoTrackAtAll) {
-          const err = new MediaError(
-            "NO_AUDIO_VIDEO_TRACKS",
-            "No audio and no video tracks are set.",
+    // Use a setTimeout(..., 0) to let a time window for an audio/video track switch when
```

Comment:
Note that I wrote time window in my test but I don't think we should have a "time window". Ideally this is easier to handle if this is all synchronous but the `noPlayableTrack` event is sent before the error is. The test should pass too.